app/uploaders/picture_uploader.rb

```diff
@@ -39,7 +42,8 @@ def extension_whitelist
   end
 
   # Override the filename of the uploaded files:
-  # Avoid using model.id or version_name here, see uploader/store.rb for details.
+  # Avoid using model.id or version_name here,
+  # see uploader/store.rb for details.
   # def filename
   #   "something.jpg" if original_filename
   # end
```

Comment:
Can you delete all the comments for this file? They're just instructions, then it'll be ready to merge! 😄